src/entities/Admin/isAdmin.ts

```diff
@@ -0,0 +1,19 @@
+import { magenta } from 'colors/safe'
+import env from 'decentraland-gatsby/dist/utils/env'
+import isEthereumAddress from 'validator/lib/isEthereumAddress'
+
+export const ADMIN_ADDRESSES = (env('ADMIN_ADDRESSES', '') || '')
+  .split(',')
+  .filter(isEthereumAddress)
+  .map((address) => address.toLowerCase())
+
+const adminAddresses = new Set(ADMIN_ADDRESSES)
+
+adminAddresses.forEach((address) => console.log('admin address:', magenta(address)))
+
+export default function isAdmin(user?: string | null | undefined) {
```

Comment:
Is this function used? 
I'd move this two lines inside the function
```
const adminAddresses = new Set(ADMIN_ADDRESSES)

adminAddresses.forEach((address) => console.log('admin address:', magenta(address)))
```

Reply:
Right, I removed the fn and left that code outside so it runs on start, and it logs the addresses like the committee ones :)